examples/README.txt

```diff
@@ -0,0 +1,9 @@
+.. _general_examples:
```

Comment:
what is it? you create new folder in root with this one, probably incorrect location

Reply:
Nope. *examples* directory is the thing for the sphinx-gallery. This line is just a reference name for this file.

Reply:
why this in the root of the repository?  Better to move it do `docs/`

Reply:
Well, I can do that. [Scikit-learn](https://github.com/scikit-learn/scikit-learn) and sphinx-gallery themselves place it in the root dir though.